content/rest/reference/apps.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Apps
+intro: 'The GitHub Apps API enables you to retrieve the information about the installation as well as specific information about GitHub Apps.'
```

Comment:
This looks very similar to the first sentence of the body. It's a great idea to take inspiration from there! Instead of mostly repeating it, though, let's move it here and remove it from the body.

Reply:
Sure thing. Thanks for the suggestions on editing.